Jellyfin.Server/Middleware/BaseUrlRedirectionMiddleware.cs

```diff
@@ -53,6 +53,12 @@ public async Task Invoke(HttpContext httpContext, IServerConfigurationManager se
             {
                 // Always redirect back to the default path if the base prefix is invalid or missing
                 _logger.LogDebug("Normalizing an URL at {LocalPath}", localPath);
+
+                if (string.IsNullOrWhiteSpace(baseUrlPrefix))
+                {
+                    baseUrlPrefix = ".";
```

Comment:
Is setting this to `.` rather than an empty string really ideal though?

Reply:
What I mean is, maybe it's better to in the next line strip out the `/` part and add an `else` here to add it back. So the empty path is truly relative.

So if there's no baseurl, you get just `_configuration[Configuration...`. If there is, you get `baseurlprefix + / + config` as is there now.

Reply:
e.g.

```
--- a/Jellyfin.Server/Middleware/BaseUrlRedirectionMiddleware.cs
+++ b/Jellyfin.Server/Middleware/BaseUrlRedirectionMiddleware.cs
@@ -54,12 +54,12 @@ namespace Jellyfin.Server.Middleware
                 // Always redirect back to the default path if the base prefix is invalid or missing
                 _logger.LogDebug("Normalizing an URL at {LocalPath}", localPath);
 
-                if (string.IsNullOrWhiteSpace(baseUrlPrefix))
+                if (!string.IsNullOrWhiteSpace(baseUrlPrefix))
                 {
-                    baseUrlPrefix = ".";
+                    baseUrlPrefix = baseUrlPrefix + "/";
                 }
 
-                httpContext.Response.Redirect(baseUrlPrefix + "/" + _configuration[ConfigurationExtensions.DefaultRedirectKey]);
+                httpContext.Response.Redirect(baseUrlPrefix + _configuration[ConfigurationExtensions.DefaultRedirectKey]);
                 return;
             }
```

Reply:
I did consider this as an alternative solution, and I think it is valid, but I have not actually tested it.
I can test it if you would like.